src/potions.ts

```diff
@@ -249,6 +253,10 @@ export function potionSetup(doEmbezzlers = false): void {
   }
 }
 
+/**
+ * Uses a Greenspan iff profitable; does not account for PYEC/LTC, or running out of adventures with the ascend flag.
+ * @param embezzlers Do we want to account for embezzlers when calculating the value of bathroom finance?
```

Comment:
`* @param {number} embezzlers Do we want to account for embezzlers when calculating the value of bathroom finance?`